lib/db/mysql/index.js

```diff
@@ -148,13 +151,20 @@ const QUERY_CODE_INSERT =
   'INSERT INTO codes (clientId, userId, email, scope, authAt, code) ' +
   'VALUES (?, ?, ?, ?, ?, ?)';
 const QUERY_TOKEN_INSERT =
-  'INSERT INTO tokens (clientId, userId, email, scope, type, token) VALUES ' +
-  '(?, ?, ?, ?, ?, ?)';
+  'INSERT INTO tokens (clientId, userId, email, scope, type, expiresAt,token)' +
+  ' VALUES (?, ?, ?, ?, ?, ?)';
```

Comment:
Is this missing a `"?"`? It seems like we added `token` above and have 7 fields, but only 6 `?`s.